README.rst

```diff
@@ -88,6 +88,15 @@ Tokens are wrapped within tuples due to the ability to specify any number of n-g
 
 Take a look at the function's docstring for information on how to use ``stopwords``, specify a ``min_length`` for tokens, and configure token output using the ``ignore_numeric`` and ``retain_casing`` parameters.
 
+By default, ``word_tokenize`` omits whitespace from the output token stream; whitespaces are rarely useful to include in a document term index.
+
+If you need to tokenize text and re-assemble an output with spacing that matches the input, you may enable the ``tokenize_whitespace`` setting.
+
+.. code-block:: python
+
+    list(textparser.word_tokenize('Conventions.  May. Differ.', tokenize_whitespace=True))
+    [('conventions',), ('  ',), ('may',), (' ',), ('differ',)]
```

Comment:
My initial idea was that whitespace tokens might be better off as a separate return value in order to make it obvious what is and is not considered whitespace (rather than having to analyse each element). 

However I now realise that its always going to be the case that the elements at even indexes will be the whitespaces and the elements at odd indexes will be words.

Might be worth explaining this in the README over here?

Reply:
@MichaelAquilina That seems like it would almost be true, but there's a catch :)

When tokenizing both puncuation and whitespace, it's not guaranteed that even-indexed elements will be whitespaces (I've added an additional test case to demonstrate this).

Reply:
Yep that's a corner case I didn't think about. Same problem applies to returning a separate return value in that case (since you can't know the order to put the two results back together). Let's leave this as is and leave the whitespace detection to the caller. We could perhaps consider adding a function such as `is_whitespace` to help out if needs be.